lib/index.d.ts

```diff
@@ -49,73 +49,50 @@ export class StateController<State> {
   unsubscribeUpdates(listener: (props: Partial<State>) => void): void;
 }
 
-declare namespace Component {
```

Comment:
removing namespace Component because if conflicts with Component (the class) declaration